spec/credit_card_validations_spec.rb

```diff
@@ -105,53 +105,60 @@
 
   describe "adding/removing brand" do
 
+    describe "adding rules" do
 
-    let(:voyager_number) {
-      '869926275400212'
-    }
-
-    it "should validate number as voyager" do
-      CreditCardValidations::Detector.add_brand(:voyager, {length: 15, prefixes: '86'})
-      detector(voyager_number).valid?(:voyager).must_equal true
-      detector(voyager_number).voyager?.must_equal true
-      detector(voyager_number).brand.must_equal :voyager
-    end
-
-
-    describe "Add voyager rule" do
-      before do
-        CreditCardValidations::Detector.add_brand(:voyager, {length: 15, prefixes: '86'})
-      end
+      let(:voyager_number) { '869926275400212' }
 
       it "should validate number as voyager" do
-
+        CreditCardValidations::Detector.add_brand(:voyager, { length: 15, prefixes: '86' })
         detector(voyager_number).valid?(:voyager).must_equal true
         detector(voyager_number).voyager?.must_equal true
         detector(voyager_number).brand.must_equal :voyager
-
       end
 
-      describe "Remove voyager rule" do
+      describe "Add voyager rule" do
         before do
-          CreditCardValidations::Detector.delete_brand(:voyager)
+          CreditCardValidations::Detector.add_brand(:voyager, { length: 15, prefixes: '86' })
```

Comment:
Line is too long. [93/80]<br>Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.